dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/CoreFeature.kt

```diff
@@ -65,14 +65,18 @@ internal object CoreFeature {
     internal lateinit var dataUploadScheduledExecutor: ScheduledThreadPoolExecutor
     internal lateinit var dataPersistenceExecutorService: ExecutorService
 
+    init {
+        System.loadLibrary("native-lib")
```

Comment:
`native-lib` is a really generic name, and you also use the same name in the sample. I'd rather find a more explicit name, eg `dd-native-android-lib`.

Reply:
Like I mentioned before Xavier, this is just a playground, nothing in this PR will probably remained like that, was just for you to have a first contact with the NDK layer and to exchange ideas about how and where we need to plug - in